resources/product.py

```diff
@@ -58,12 +58,14 @@ def get(cls, product_id):
     def post(cls, current_user=None):
         """Handles the post request and creates a new product"""
         body = clean_arguments(product_args)
+        body["organizer_username"] = current_user.username
+        body["status"] = "Funding"
         product_id = ObjectId()
         new_product = product_model(**body, _id=product_id)
         new_group = group_model(
             product_id=product_id,
             organizer_id=request.json.get("user_id"),
-            status="Funding",
+            user_id=[request.json.get("user_id")],
```

Comment:
Is the frontend not gonna return an array of user_ids so in this case the array is redundant?
i.e. [ [user_id] ]
```suggestion
            user_id=request.json.get("user_id"),
```
Otherwise LGTM